Failing test in AIFFTest.test_read_audio_properties

The test suite is currently failing across both Python2 and Python3, on multiple Linux distributions:

```
.........F........................................................................................................................................................................................................................................................................................................................................................................................................................................
======================================================================
FAIL: test_read_audio_properties (test.test_mediafile.AIFFTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/apetresc/src/arch/python-mediafile/src/python3/test/test_mediafile.py", line 441, in test_read_audio_properties
    self.assertEqual(getattr(mediafile, key), value)
AssertionError: 16 != 0

----------------------------------------------------------------------
Ran 434 tests in 0.663s

FAILED (failures=1)
```

This is blocking the Arch Linux AUR package (among, probably, many others) for everyone.

--- a/setup.py
+++ b/setup.py
@@ -100,7 +100,7 @@ setup(
 
     install_requires=[
         'six>=1.9',
-        'mutagen>=1.33',
+        'mutagen>=1.43',
     ] + (['enum34>=1.0.4'] if sys.version_info < (3, 4, 0) else []),
 
     tests_require=[

--- a/test/test_mediafile.py
+++ b/test/test_mediafile.py
@@ -930,7 +930,7 @@ class AIFFTest(ReadWriteTestBase, unittest.TestCase):
         'bitrate': 705600,
         'format': u'AIFF',
         'samplerate': 44100,
-        'bitdepth': 0,
+        'bitdepth': 16,
         'channels': 1,
     }
 

